src/tests/Directory.Build.props

```diff
@@ -135,4 +135,11 @@
     <NuGetTargetMoniker>$(NetCoreAppCurrentToolTargetFrameworkMoniker)</NuGetTargetMoniker>
     <NuGetTargetMonikerShort>$(NetCoreAppToolCurrent)</NuGetTargetMonikerShort>
   </PropertyGroup>
+
+  <!-- Set Test Wrapper running host OS -->
+  <PropertyGroup>
+    <TestWrapperTargetsWindows>false</TestWrapperTargetsWindows>
+    <TestWrapperTargetsWindows Condition=" ('$(TargetsWindows)' != '' And '$(TargetsWindows)' ) OR ('$(TargetOS)' == 'Android' And '$(TargetArchitecture)' == 'arm64' )">true</TestWrapperTargetsWindows>
```

Comment:
Do we use ".cmd" scripts on arm64 android?

Reply:
I think the trick is that, while traditional CoreCLR builds deal with a pair of platforms (hosting / targeting), for Mono device testing there's a triplet of platforms - host, test driver, target. Arm64 Android tablets use Windows test drivers so at that level yes, they use ".cmd" scripts.